BDArmory/Distribution/GameData/BDArmory/Localization/UI/ja.cfg

```diff
@@ -0,0 +1,1194 @@
+        #LOC_BDArmory_AIWindow_DefAlt = AIはアイドル状態になるとこれに戻ります
+        #LOC_BDArmory_AIWindow_MinAlt = AIはこれを超えようとします
+        #LOC_BDArmory_AIWindow_MaxAlt = AIはこれ以下に留まろうとする
+        #LOC_BDArmory_AIWindow_maxSpeed = AI はこの対気速度以下に留まります
+        #LOC_BDArmory_AIWindow_takeoff = Speed AI は離陸時にピッチ入力を開始します
+        #LOC_BDArmory_AIWindow_minSpeed = この速度を下回ると、AI はエネルギーを回復しようとします
+        #LOC_BDArmory_AIWindow_atkSpeed = 対地攻撃速度
+        #LOC_BDArmory_AIWindow_idleSpeed = 非戦闘巡航速度
+        #LOC_BDArmory_AIWindow_ABPriority = アフターバーナーを有効にするためのしきい値を変更します
+        #LOC_BDArmory_AIWindow_ABOverrideThreshold_Context = この速度以下でフルスロットルの場合はアフターバーナーを強制的に使用します
+        #LOC_BDArmory_AIWindow_LSSL = 低速制限以下の制限制御
+        #LOC_BDArmory_AIWindow_LSLS = AI はこの速度以下では低速リミッターを使用します
+        #LOC_BDArmory_AIWindow_HSSL = 高速制限を超える制御を制限
+        #LOC_BDArmory_AIWindow_HSLS = AI はこの速度を超えると上限に完全に制限されます
+        #LOC_BDArmory_AIWindow_ASLF = 高度に基づいてステア制限を増減する係数
+        #LOC_BDArmory_AIWindow_ASLA = ステア制限の減少/増加を開始する高度
+        #LOC_BDArmory_AIWindow_WPPreRoll = ウェイポイントに到達する前にローリングを開始する
+        #LOC_BDArmory_AIWindow_WPYawAuth = ウェイポイントに近づくときのヨー応答を増加します
+        #LOC_BDArmory_AIWindow_bankLimit = 最大ロール角
+        #LOC_BDArmory_AIWindow_GForce = マニューバはこの G 制限を超えないよう努めます
+        #LOC_BDArmory_AIWindow_AoA = マニューバ AoA はこの AoA を超えないよう努めます
+        #LOC_BDArmory_AIWindow_AoAPostStall = この AoA を超えた失速後に飛行モードを切り替える
+        #LOC_BDArmory_AIWindow_ImmelmannTurnAngleContext = クラフトはこのコーン内のターゲットを狙うためにピッチアップするだけです
+        #LOC_BDArmory_AIWindow_MinEvade = AI が攻撃を回避する最小時間
+        #LOC_BDArmory_AIWindow_EvExNonlin = 回避・伸長時の振動の強さ
+        #LOC_BDArmory_AIWindow_evadeDist = この距離内に砲火が入ってきた場合は回避します
+        #LOC_BDArmory_AIWindow_evadetimeDist = 射撃を受けて回避を引き起こすまでの最小時間
+        #LOC_BDArmory_AIWindow_ColDist = この距離内で接近してくる航空機を回避してください
+        #LOC_BDArmory_AIWindow_ColTime = AI が衝突を探すのは何秒前か
+        #LOC_BDArmory_AIWindow_ColStrength = AI が到来する航空機からどれだけ困難に離脱するか
+        #LOC_BDArmory_AIWindow_standoff = AI がターゲットに近づこうとする距離
+        #LOC_BDArmory_AIWindow_ExtendAngleAirToAir_Context = 伸長時の希望上昇角度
+        #LOC_BDArmory_AIWindow_ExtendDistanceAirToAir_Context = 空対空距離の延長
+        #LOC_BDArmory_AIWindow_ExtendDistanceAirToGroundGuns_Context = 空対地距離の延長 (砲)
+        #LOC_BDArmory_AIWindow_ExtendDistanceAirToGround_Context = 空対地距離の延長
+        #LOC_BDArmory_AIWindow_Extendvel = ターゲットが遅すぎて方向を向くことができない場合に設定します
+        #LOC_BDArmory_AIWindow_ExtendAngle = ターゲットが回転半径の外側にあるときに設定します
+        #LOC_BDArmory_AIWindow_ExtendDist = ターゲットが近すぎて振り向くことができない場合に設定します
+        #LOC_BDArmory_AIWindow_ExtendAbortTimeContext = 延長中に距離を稼げなかった場合は時間を中止します。
+        #LOC_BDArmory_AIWindow_terrainMin = 理想的な機体の向きを実現する回転半径乗数
+        #LOC_BDArmory_AIWindow_terrainMax = 反転したクラフトの向きの回転半径乗数
+        #LOC_BDArmory_AIWindow_InvertedTerrainAvoidanceCriticalAngleContext = 逆地形回避またはローリングファーストの臨界角
+        #LOC_BDArmory_AIWindow_TerrainAvoidanceControlSurfaceDeploymentTimeContext = コントロール サーフェスの展開にかかる時間の見積もり

```

Comment:
`#LOC_BDArmory_AIWindow_TerrainAvoidanceControlSurfaceDeploymentTimeContext` is now `#LOC_BDArmory_AIWindow_TerrainAvoidanceVesselReactionTimeContext = Estimate of time required to setup for optimal turning`